.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-    "javascript.validate.enable": false
+    "javascript.validate.enable": false,
+    "eslint.enable": true
```

Comment:
And remove eslint.enable from settings.json, it is unnecessary.
I hope you have the Flow and ESLint plugins?